scripts/pi-hole/js/queries.js

```diff
@@ -115,6 +115,11 @@ $(function () {
         isCNAME = false,
         regexLink = false;
 
+      // accompanies Store domainlist IDs for blocked/permitted queries FTL PR 1409
+      if (data.length > 9 && data[9] > 0) {
```

Comment:
We should check if the `data[9]` field is numeric - could also be a CNAME domain or empty- before checking if it is greater than zero:

```suggestion
      if (data.length > 9 && is_numeric(data[9]) && data[9] > 0) {
```

Note: I am on a business trip and haven't been able to actually test this suggestion.